samples/user-interface/predictiveback/build.gradle.kts

```diff
@@ -19,6 +19,7 @@
 plugins {
     id("com.example.platform.sample")
     alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.compose.compiler)
```

Comment:
why do we need to add this?

Reply:
This is a new requirement for Kotlin 2.0 that the Kotlin Compiler needs to be added